data/com.github.tchx84.Flatseal.appdata.xml.in

```diff
@@ -26,7 +26,8 @@
     </screenshot>
   </screenshots>
   <update_contact>martin.abente.lahaye@gmail.com</update_contact>
-  <developer_name>Martin Abente Lahaye</developer_name>
+  <!-- TRANSLATORS: Don't translate this text -->
```

Comment:
You need to run `ninja flatseal-pot` to actually update the template (pot). And it will take comments for `name` and `id`, so please add comments of these as well :)

Reply:
I didn't update it manually, I used `ninja flatseal-pot` as you suggested. Hmm, I just noticed that you were right, the strings were not ignored by gettext. I will add comments to them.

Reply:
I added comments to the name tags (in AppData and desktop files). The `id` tags seem to get ignored by gettext, no need to do anything with them.

Reply:
Alright then!